test/src/test/java/org/corfudb/integration/CorfuReplicationLargeTxIT.java

```diff
@@ -115,17 +118,30 @@ private void testTxChunking(int numEntriesToWrite,
         int totalStreamingUpdates =
             expectedStreamingUpdatesPerTable * mapNameToMapStandby.size();
 
+        // MAX_SNAPSHOT_ENTRIES_APPLIED = 1.  So entries to the ProtobufDescriptor table will be applied in a batch
+        // of 1.  Hence, the number of transactions made = number of entries received from the Source Cluster(1 TX
+        // per entry).
+        int numExpectedTxOnProtobufDescriptorTable =
```

Comment:
You should validate that numExpectedTxOnProtobufDescriptorTable != 1, otherwise, we wouldn't be sure if our fix is working, cause if there is only 1 entry then naturally only one entry would be applied.